scipy/stats/tests/test_discrete_distns.py

```diff
@@ -29,6 +29,15 @@ def test_hypergeom_logpmf():
     assert_almost_equal(hypergeom_logpmf, bernoulli_logpmf, decimal=12)
 
 
+def test_nhypergeom_logpmf():
+    # test with hypergeom
+    N, K, r = 45, 13, 8
+    k = 6
+    NHG = nhypergeom._pmf(k, N, K, r)
+    HG = hypergeom._pmf(k, N, K, k+r-1) * (N - K - (r-1)) / (N - (k+r-1))
+    assert_almost_equal(HG, NHG, decimal=10)
```

Comment:
Use `assert_allclose` (with an appropriate value of `rtol`) instead of `assert_almost_equal`.  This preference is noted in the docstring of `assert_almost_equal`.
```suggestion
    assert_allclose(HG, NHG, rtol=1e-10)
```

Reply:
👍